qemu/tests/cfg/blockdev_stream_forbidden_actions.cfg

```diff
@@ -30,6 +29,8 @@
     error_msg_mirror = "Need a root block node"
     error_msg_commit = "Need a root block node"
     error_msg_resize = "Device '(null)' is in use"
+    resize_err_change_ver = [9.0.0, )
+    error_msg_resize_new = Node '${top_device_node}' is busy: block device is in use by block job: stream
```

Comment:
An easy but not exactly correct way is using Host_RHEL filters

```
error_msg_resize = Node '${top_device_node}' is busy: block device is in use by block job: stream
Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8 Host_RHEL.m9:
    error_msg_resize = "Device '(null)' is in use"
```  
It's 'partially' correct because this is upstream project, we cannot consider RHEL only, but..., any other released linux could add theirs if they want to run this case, so I am OK with that :-D

Reply:
@XueqiangWei @aliang123 what do you think?

Reply:
Sorry for the late reply. I agree with you. Thanks.